### PR TITLE
Update SubjectGroupViewer README (May 2021)

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview

This is a documentation update for the Subject Group Viewer system

- A better example of the workflow configuration was added. The previous documentation was missing the "subject_group.num_columns" and "subject_group.num_rows" field that's now essential for the new /subjects/grouped endpoint
- The "Subject Groups" used by the viewer has been expanded upon.

### Status

Ready for review.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
Wait, hang on, this needs additional READMEs for the Subject Group Comparison TASK. brb.

**Comment by shaunanoordin** [comment]
## PR Update

This PR now adds READMEs for the SubjectGroupComparisonTask AND the SubjectGroup "type" of Subject. OK, I'm done for today!

File: packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+different from other images on the grid!"
+
+- This task is STRONGLY ASSOCIATED with the Subject Group Viewer. Please see `src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/README.md` for more details.
+- This task is STRONGLY ASSOCIATED with the Subject Group "type" of Subjects. Please see `src/store/SubjectGroup/README.md` for more details.
+
+The Subject Group Comparison Task was originally created by @shaunanoordin on Mar 2020, for the SURVOS project.
+
+## Data Models
+
+Simple Dropdown task data structure, example:
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/README.md:14]: Wrong task name here?

**Review by eatyourgreens — APPROVED** [review]
I think the task name is wrong in the task Readme. Otherwise, looks good!

File: packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/README.md (hunk 1 above)
Comment (shaunanoordin) [reply packages/lib-classifier/src/plugins/tasks/SubjectGroupComparisonTask/README.md:14]: Whoops! Thanks for spotting this, I'll fix it right away.

**Comment by shaunanoordin** [comment]
😃 👍

**Pull request merged**